app/core/components/FormInput/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Stack } from "@mui/material";
+import { useField } from "remix-validated-form";
+import { FormHelperText } from "@mui/material";
+
+type FormInputProps = {
+  name: string;
+  label: string;
+  isRequired?: boolean;
+  value?: string;
+  setUrlWithError: any;
+};
+
+export const FormInput = ({
+  name,
+  label,
+  isRequired,
+  value,
+  setUrlWithError,
+  ...rest
+}: FormInputProps) => {
+  const { getInputProps, error } = useField(name);
+
+  if (error === "Invalid url") {
+    setUrlWithError(true);
+  } else {
+    setUrlWithError(false);
+  }
+
+  return (
+    <Stack direction="column-reverse" alignItems="center">
```

Comment:
Since the input is hidden, maybe we don't need to use a Stack component here, right?

Reply:
well I need it at least to align the error text 😬